src/bundle/Resources/views/content/tab/url/checkbox.html.twig

```diff
@@ -0,0 +1,8 @@
+{%- block checkbox_widget -%}
+    <div class="ez-field-edit--ezboolean">
+        <label class="ez-data-source__label{% if checked %} is-checked{% endif %}">
+            <span class="ez-data-source__indicator"></span>
+            <input type="checkbox" {{ block('widget_attributes') }}{% if value is defined %} value="{{ value }}"{% endif %}{% if checked %} checked="checked"{% endif %} />
```

Comment:
If you are using `widget_attributes` block you should inject `value=""` and `checked=""` attributes into the `attr` variable because if someone decided to do so it will display `value` and `checked` twice.